hookman/plugin_config.py

```diff
@@ -23,17 +23,22 @@ class PluginInfo(object):
     description = attrib(type=str, default="Could not find a description", init=False)
     email = attrib(type=str, init=False)
     hooks_implemented = attrib(type=list, init=False)
-    name = attrib(type=str, init=False)
+    plugin_caption = attrib(type=str, init=False)
```

Comment:
TBH it is strange to prefix `name` and `caption` for `plugin`, given that we are already in the class `PluginInfo`. For example, why not `plugin_description`, `plugin_email`, `plugin_version`, and so on?

Reply:
I created a new PR implementing the changes request in our discussion.
https://github.com/ESSS/hookman/pull/64

Reply:
> I created a new PR implementing the changes request in our discussion.
> #64

Could we discuss this approach?